marshal/consumer.go

```diff
@@ -622,28 +630,37 @@ func (c *Consumer) Terminate(release bool) bool {
 // consumer. It should be relevent only when ClaimEntireTopic is set
 func (c *Consumer) GetCurrentTopicClaims() (map[string]bool, error) {
 	if !c.options.ClaimEntireTopic {
-		err := fmt.Errorf("GetCurrentTopicClaims is only relevent when ClaimEntireTopic is set")
-		log.Error(err.Error())
-		return nil, err
+		return nil, errors.New(
+			"GetCurrentTopicClaims requires options.ClaimEntireTopic be set")
+	}
+
+	claimedTopics := make(map[string]bool)
+	if c.Terminated() {
```

Comment:
Is this necessary?

If we're properly terminated then I would think this `range` on `c.partitions` would come to the same conclusion. Namely, not adding anything to the `claimedTopics` map. However, if we are in some bogus state where we are terminated with claims perhaps it is best not to cover it up?